.github/workflows/dependency-scan.yml

```diff
@@ -3,6 +3,8 @@ name: Check dependencies
 on:
   push:
     branches: '*'
+  pull_request:
+    branches: '*'
```

Comment:
I guess so, for most pull requests. 

But if I made a PR against a branch from another PR (e.g. https://github.com/THLfi/koronavilkku-android/pull/28), would that mean it wouldn't build?

What are the benefits of restricting it to `trunk`?

Reply:
I just think it's a good habit to be explicit when possible. Though, you are correct that it won't run if pr is made against some other branch than trunk, there is probably not much benefits to run action on that pr (made against some other branch than trunk), because currently we are following principles of trunk based development i.e. release branches are always made from trunk. Feature branches should also always made from trunk. However, even if there is some pr created from some arbitrary branch, it's still enough to run checks when that another branch is pushed into branch made from trunk.

Reply:
Okay, updated!